Content.IntegrationTests/Tests/Announcers/AnnouncerAudioTest.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.Announcements.Systems;
+using Content.Server.StationEvents;
+using Content.Shared.Announcements.Prototypes;
+using Robust.Client.ResourceManagement;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC.Exceptions;
+using Robust.Shared.Localization;
+using Robust.Shared.Prototypes;
+using Serilog;
+
+namespace Content.IntegrationTests.Tests.Announcers;
+
+/// <summary>
+///     Checks if every station event wanting the announcerSystem to send audios has a sound file
```

Comment:
```suggestion
///     Checks if every station event using the announcerSystem has a valid audio file associated with it.
```

Call me stupid but this took me a couple of re-reads to understand, just slightly iffy wording.